graphql/src/Mu/GraphQL/Server.hs

```diff
@@ -59,18 +57,6 @@ instance A.FromJSON ValueConst where
       toObjFld :: (T.Text, ValueConst) -> ObjectFieldG ValueConst
       toObjFld (k, v) = ObjectFieldG (coerce k) v
 
-type GraphQLApp p pname ss qmethods mmethods hs chn qr mut qanns manns =
```

Comment:
Since you moved this to another file, you can now safely remove `ConstraintKinds` on https://github.com/higherkindness/mu-haskell/pull/134/files#diff-c037c2c61f81962ca06b553051605319R1